requirements.txt

```diff
@@ -58,6 +58,6 @@ six==1.10.0
 git+https://github.com/bartTC/django-attachments.git
 git+https://github.com/CSnap/django-likes.git
 git+https://github.com/CSnap/django_teams.git
+git+https://github.com/RuofanLiu/SGIS-backend.git
```

Comment:
settings.py is updated, in addition, please see the update in the backend, if it is ok, please accept the PR so that I could change this from my own repo to CSnap.